dev/io.openliberty.data/bnd.bnd

```diff
@@ -16,6 +16,9 @@ bVersion=1.0
 javac.source: 17
 javac.target: 17
 
+tested.features: \
```

Comment:
I don't think this project has any FAT tests.

Reply:
You're correct, that was meant to be an update to `dev/io.openliberty.data.internal_fat/bnd.bnd` (which is also now correctly updated).